### PR TITLE
sigin form

### PR DESCRIPTION
**Pull request by havvarslan**
Login and Logout displayed
Login form displayed
Get error when the info is not convenient

### CODE REVIEW

**Comment by islam95** [comment]
You have merge conflict here, can you fix it please?

**Comment by islam95** [comment]
You need to pull upstream again. You still have merge conflicts :)

**Comment by havvarslan** [comment]
> You need to pull upstream again. You still have merge conflicts :)

**Comment by havvarslan** [comment]
How about now?

File: client/src/components/Home.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import { Header } from "semantic-ui-react";
+import Login from "./Login";
 
 const Home = () => {
   return (
     <div>
-      <Header as="h1">Scot Phamy Talent</Header>
+      {/* <Header as="h1">Scot Phamy Talent</Header> */}
```
[the remark this answers is not in the document]
Comment (havvarslan) [reply client/src/components/Home.js]: Actually this is not my part:)

File: client/src/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import { signApi } from "../api/auth";
+
+export default class Login extends Component {
+  state = {
+    email: "",
+    password: "",
+    error: null
+  };
+
+  // handleSubmit = event => {
```
[the remark this answers is not in the document]
Comment (havvarslan) [reply client/src/components/Login.js]: I will

File: client/src/index.js [hunk 3]
```diff
@@ -16,18 +17,27 @@ const Routes = () => {
     <Container>
       <Router>
         <NavBar />
+        <Grid width={15} centered>
```
Comment (islam95) [inline client/src/index.js]: You need to leave incoming changes in this file as your team mate changed the way the routes work in you project

File: client/src/index.js [hunk 4]
```diff
@@ -16,18 +17,27 @@ const Routes = () => {
...
+          <div style={{ margin: "20px" }}>
+            <Switch>
+              <Route path="/" exact component={Home} />
+              <Route path="/about" exact component={About} />
+              <Route path="/status" exact component={Status} />
 
-        <div style={{ margin: "20px" }}>
-          <Route path="/" exact component={Home} />
-          <Route path="/about" exact component={About} />
-          <Route path="/status" exact component={Status} />
```
Comment (islam95) [inline client/src/index.js]: You can just put your route inside here

**Pull request merged**